content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,240 @@
+
+# verify installation
+jx --version
+{{</highlight>}}
+        
+        </div>
+        <div class="tab-pane fade" id="linux">
+        
+        &nbsp;  
+        
+      
+{{<highlight sh>}}
+# Download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+
+# if you don't have `jq` installed:
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent "https://github.com/jenkins-x/jx/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+
+# Move binary to your path
+sudo mv jx /usr/local/bin
+
+# verify installation
+jx --version
+{{</highlight>}}
+      
+        </div>
+        <div class="tab-pane fade" id="windows">
+        &nbsp;
+      
+        <h3> Install Chocolatey using an Admin</h3>
+      
+      
+      1. Right-click menu:Start\[Command Prompt (Admin)\].
+      <br><br>
+      2. At the shell prompt, execute a `powershell.exe` command to download
+          and install the `choco` binary and set the installation path so that
+          the binary can be executed:
+
+{{<highlight sh>}}
+    @"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+{{</highlight>}}
+
+      3.  Install Jenkins X using Chocolatey:
+      
+{{<highlight sh>}}
+  choco install jenkins-x
+{{</highlight>}}
+      
+      Upgrading Jenkins X via Chocolatey
+      
+{{<highlight sh>}}
+choco upgrade jenkins-x
+{{</highlight>}}
+      
+        </div>
+      </div>
+    </div>
+  </div>   
+  <p>&nbsp;</p>
+  <div class="row">
+  <div class="col col-lg-12">
+    <p>&nbsp;</p>
+    <h2>Step 3: Install Jenkins X on your cluster</h2>
+    
+    <p>You have provisioned your Kubernetes cluster and the CLI.  You are ready to install Jenkins X on top of the cluster.  To do that, you will use the <strong>JX Boot</strong> CLI command.</p>
+    <div class="alert alert-dismissible alert-primary"> 
+      <strong>What is JX Boot?</strong> With JX Boot you Install, configure or upgrade Jenkins X via GitOps and a Jenkins X Pipeline.  This pipeline resides within your
+      config git repository, also known as a <a class="alert-link" href="/about/concepts/components/#permanent-environments">permanent environment</a>
+    </div>
+  {{<highlight bash>}}
+    # the parameter here, is the requirements yaml file
+    # which was saved in the same directory you executed terraform apply.
+    $ jx boot -r jx-requirements.yml
+  {{</highlight>}}
+  <p>Get more details on <a href="/docs/getting-started/setup/boot/">JX Boot</a>  </p>
+  </div>
+  </div>
+  
+<div class="row">
```

Comment:
+1, I like the box at the bottom better. Less clutter at the top. Maybe we can add a decided though or create more space so that the separation to Step 3 becomes clearer.

Reply:
Agreed on separating the major 3 steps a bit more, let me work on that now!